changes/5791.changed

```diff
@@ -0,0 +1 @@
+Allow Device Primary IP can now be selected from Child Modules.
```

Comment:
Perhaps `.added` rather than `.changed` since it's net-new functionality. Perhaps:

```suggestion
Added support for specifying a Device's Primary IP from an interface on a child Module.
```